### PR TITLE
alan progress week 13

### PR DESCRIPTION
**Pull request by alanssitis**

### CODE REVIEW

**Review by WDobert — CHANGES_REQUESTED** [review]
Remove the json file that got committed

**Review by WDobert — APPROVED** [review]

File: web/src/pages/team/achungma.md [hunk 1]
```diff
@@ -3,9 +3,45 @@ layout: ../../layouts/MDLayout.astro
...
 
-# Progress Report for Alan\
+# Progress Report for Alan
 
-## Weel 12
+## Week 13
+
+**Date:** 2023-11-17 \
+**Week Hours:** 13 \
+**Total Hours:** 143
```
Comment (WDobert) [inline web/src/pages/team/achungma.md:12]: Can't suggest changes on mobile, but hours should be 153

**Review by WDobert — CHANGES_REQUESTED** [review]

**Review by benjamin-owen — APPROVED** [review]

File: web/src/pages/team/achungma.md (hunk 1 above)
Comment (benjamin-owen) [reply web/src/pages/team/achungma.md:12]: :nerd_face:

**Review by WDobert — APPROVED** [review]

**Review by tream2001 — APPROVED** [review]
It's after midnight, so I ain't checking

**Pull request merged**